src/ex1/index.html

```diff
@@ -1,14 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
+
```

Comment:
## General notes
- next time please add a picture or even a gif demonstrating the product
- move all your files outside of this folder to be directly under src folder because this is going to be a rolling exercise and we don't want it to be on ex1 folder

## To keep
- very cool design and creative empty state
- great additions: deadline, complete

## To improve
- The sorting functionality doesn't work
- Read about the `trim()` method and use it when inserting a new task
- Although it's not an explicit requirement, we expected to have a tasks counter in the bottom of the tasks list